ext/mysqlnd/mysqlnd_enum_n_def.h

```diff
@@ -130,6 +130,7 @@
 #define CR_INVALID_PARAMETER_NO	2034
 #define CR_INVALID_BUFFER_USE	2035
 #define CR_LOAD_DATA_LOCAL_INFILE_REJECTED 2068
+#define ER_CLIENT_INTERACTION_TIMEOUT 4031
```

Comment:
What does `ER_` stand for?

Reply:
This is copy paste from mysql-server, will change to `CR_CLIENT_INTERACTION_TIMEOUT`